Sources/TLS/Certificates.swift

```diff
@@ -41,25 +41,26 @@ public enum Certificates {
     }
 
     public static var defaults: Certificates {
-        if let system = system {
-            return system
-        } else {
-            return openbsd
-        }
+		if let system = system { return system }
+		if let openbsd = openbsd { return openbsd }
+		fatalError("No system certificates found")
     }
 }
 
 extension Certificates {
-    public static var openbsd: Certificates {
+    public static var openbsd: Certificates? {
```

Comment:
We can't change this API since it's public (that would require a major revision).